tests/fixtures/mockbin/mockbin

```diff
@@ -45,10 +48,12 @@ if prog == 'tito':
     parser_release.add_argument('-y', action='store_const', dest='output', const='Created task: 1234')
     parser_release.add_argument('--arg')
     parser_build = subparsers.add_parser('build')
-    parser_build.add_argument('--srpm', action='store_const', dest='output', const='Wrote: hello.src.rpm')
+    parser_build.add_argument('--srpm', action='store_const', dest='output', const="Wrote: {}".format(_FAKE_SRPM_PATH))
     parser_build.add_argument('--scl', action='store')
     parser_build.add_argument('--offline', action='store_true')
     args = parser.parse_args()
+    if 'output' in args and re.compile(r'Wrote').search(args.output):
```

Comment:
I think you can just write `if 'output' in args and _FAKE_SRPM_PATH in args.output`